ultravox/data/datasets.py

```diff
@@ -1129,6 +1130,7 @@ def __init__(
         stop_strategy: StopStrategy = StopStrategy.LAST_EXHAUSTED,
         seed: Optional[int] = 42,
         static: bool = False,
+        using_epochs: bool = False,
```

Comment:
The name doesn't make much sense.
Maybe you want something like `use_proportional_weights` or something similar?